packages/app/source/server/projections/todos-projection.js

```diff
@@ -1,7 +1,7 @@
 Space.eventSourcing.Projection.extend('Todos.TodosProjection', {
 
-  collections: {
-    todos: 'Todos.TodoLists'
+  dependencies: {
```

Comment:
… which I just realised there is no abstract `ProjectionRepository` class, but the point still stands in that the ProjectionRebuilder should just be calling a method to coordinate the rebuild, therefore making it possible to support any technology